docs/database/elasticsearch.md

```diff
@@ -31,12 +32,13 @@ If the endpoint id is not available, the span name SHOULD be the `http.request.m
 | [`url.full`](/docs/attributes-registry/url.md) | string | Absolute URL describing a network resource according to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) [3] | `https://localhost:9200/index/_search?q=user.id:kimchy` | `Required` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 | [`db.elasticsearch.path_parts.<key>`](/docs/attributes-registry/db.md) | string | A dynamic value in the url path. [4] | `db.elasticsearch.path_parts.index=test-index`; `db.elasticsearch.path_parts.doc_id=123` | `Conditionally Required` when the url has dynamic values | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`server.port`](/docs/attributes-registry/server.md) | int | Server port number. [5] | `80`; `8080`; `443` | `Conditionally Required` [6] | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
-| [`db.elasticsearch.cluster.name`](/docs/attributes-registry/db.md) | string | Represents the identifier of an Elasticsearch cluster. | `e9106fc68e3044f0b1475b04bf4ffd5f` | `Recommended` [7] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`db.elasticsearch.node.name`](/docs/attributes-registry/db.md) | string | Represents the human-readable identifier of the node/instance to which a request was routed. | `instance-0000000001` | `Recommended` [8] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`db.query.text`](/docs/attributes-registry/db.md) | string | The request body for a [search-type query](https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html), as a json string. | `"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"` | `Recommended` [9] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| [`network.peer.address`](/docs/attributes-registry/network.md) | string | Peer address of the database node where the operation was performed. [10] | `10.1.2.80`; `/tmp/my.sock` | `Recommended` | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| [`db.collection.name`](/docs/attributes-registry/db.md) | string | The index or data stream against which the query is executed. [7] | `my_index`; `index1, index2` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.elasticsearch.cluster.name`](/docs/attributes-registry/db.md) | string | Represents the identifier of an Elasticsearch cluster. | `e9106fc68e3044f0b1475b04bf4ffd5f` | `Recommended` [8] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
can this be replaced by `db.namespace`?

Reply:
Yes. I changed it.

The only downside I can think of is that prior to this we had `db.elasticsearch.cluster.name` and `db.elasticsearch.node.name` (<- which we almost deprecated to use `db.instance.id`) - those are related fields. Now it changes, so we only have `db.elasticsearch.node.name`, but the cluster is in `db.namespace`.

Question is what's our preference? 
- To follow general attributes and accept that the cluster is stored in the general `db.namespace` attribute, but the instance is in the Elasticsearch specific `db.elasticsearch.node.name`. I'd say this would prefer consistency across the overall db attributes.
- To prefer consistency within Elasticsearch and have `db.elasticsearch.node.name` and `db.elasticsearch.cluster.name`, but loose some consistency with the overall db attributes on the cluster name.

I'd go with the 1. one (that's the current status of the PR).

https://github.com/open-telemetry/semantic-conventions/pull/972 is related. Without that we'd have a good story on these attributes and both cluster and instance id would be stored in a general, non-elastic searchspecific attribute.